keramik/src/analysis.md

```diff
@@ -8,6 +8,7 @@ To analyze the results of a simulation first copy the metrics-TIMESTAMP.parquet
 First restart opentelemetry-0 pod so it writes out the parquet file footer.
 
     kubectl delete pod opentelemetry-0
+    sleep 5 # Give pod time to restart
```

Comment:
Just a note, you can use the api the know when the pod is ready
```
kubectl wait --for=condition=Ready pod/opentelemetry-0
```

Reply:
Thanks, will add that in the next round of book changes